misc/drone-scripts/setup.sh

```diff
@@ -14,25 +14,28 @@
 # limitations under the License.
 
 echo "Resetting testbed"
-govc snapshot.revert -vm $ESX_6_0 init
 govc snapshot.revert -vm $ESX_6_5 init
+govc snapshot.revert -vm $ESX_6_0 init
 
 echo "Waiting for revert to complete";
 
-echo ESX 6.0
-until govc vm.ip $ESX_6_0
+echo ESX 6.5
+until govc vm.ip $ESX_6_5
 do
     echo "Waiting for revert to complete";
     sleep 1;
 done
 
-echo ESX 6.5
-until govc vm.ip $ESX_6_5
+echo ESX 6.0
+until govc vm.ip $ESX_6_0
 do
     echo "Waiting for revert to complete";
     sleep 1;
 done
 
 echo "Reset complete"
 
+sleep 5;
+echo "Resume testing"
```

Comment:
It seems you need to fix the commit .. please squash correctly as @msterin mentioned previously.